### PR TITLE
[v51] Stac 17819 Monitor docs for 51

### PR DESCRIPTION
**Pull request by aacevedoosorio**
Documentation for:

* How to get ID of monitor function for logging
* Kubernetes pod where monitor / monitor function logs can be found
* Execution type of monitor functions. (synchronous or asynchronous)
* explain the use of external tool to reformat and add function script to STJ
* Enable/Disable monitors

### CODE REVIEW

File: develop/developer-guides/custom-functions/monitor-functions.md [hunk 1]
```diff
@@ -467,6 +467,74 @@ After the function has been uploaded, it will be generally available for any mon
 
 ➡️ [Learn how to create a custom monitor that utilizes an existing monitor function](../monitors/create-custom-monitors.md)
 
+In particular for monitor functions the stj `format` can be challenging to work with given the property `scrip` of type `ScriptFunctionBody` where the value is a groovy script, so whenever an update is needed on such a value an option can be to use an external tool as [yq](https://github.com/mikefarah/yq) to get a more friendly formatting. For example, given the monitor function example above we could format it with yq :
```
Comment (craffit) [inline develop/developer-guides/custom-functions/monitor-functions.md:470]: `scrip` typo

Comment (craffit) [reply develop/developer-guides/custom-functions/monitor-functions.md:470]: would be good to quote `yq`

File: develop/developer-guides/monitors/monitor-stj-file-format.md [hunk 2]
```diff
@@ -53,6 +55,8 @@ The supported fields are:
 - **remediationHint** - a short, markdown-enabled hint displayed whenever the validation rule represented by this monitor triggers and results in an unhealthy state.
 - **function** - the specific monitor function to use as the basis of computation for this monitor. For more details see [function](#function).
 - **arguments** - lists concrete values that are to be used for parameters in the monitor function invocation. For more details and descriptions of commonly used parameters, see [arguments](#arguments).
+- **status** - either ENABLED|DISABLED. Dictates if the monitor will be running and producing health states.
```
Comment (craffit) [inline develop/developer-guides/monitors/monitor-stj-file-format.md:58]: is this argument optional?

Comment (craffit) [reply develop/developer-guides/monitors/monitor-stj-file-format.md:58]: i think it is (described below), i think a short summary of the optionality would be good herer

Comment (gggina) [reply develop/developer-guides/monitors/monitor-stj-file-format.md]: updated

**Pull request merged**